alts/src/main/java/io/grpc/alts/AltsContext.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2018 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.alts;
+
+import com.google.common.annotations.VisibleForTesting;
+import io.grpc.alts.internal.AltsInternalContext;
+import io.grpc.alts.internal.SecurityLevel;
+
+/** {@code AltsContext} contains security-related information on the ALTS channel. */
+public class AltsContext {
```

Comment:
Annotate the new classes with `@ExperimentalApi`.

Reply:
Done. Created https://github.com/grpc/grpc-java/issues/7864. Could you also give it the experimental API label?

Reply:
I've given it the label and moved it to a milestone.